extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -245,6 +247,7 @@ public function getAllMembersData() {
 		$currentUser = $this->getUser();
 		$allMembers = $this->usersModel->getAllContributors( $currentUser->getId() );
 		$allMembers = $this->addTimeAgoDataDetail( $allMembers );
+		$allMembers = $this->applyBadges( $allMembers );
```

Comment:
same as above, I don't like that we have the same name for members array in 3 different states